src/it/resources-noncompilable/org/checkstyle/suppressionxpathfilter/packageannotation/SuppressionXpathRegressionPackageAnnotationOne.java

```diff
@@ -0,0 +1,6 @@
+//non-compiled with javac: non-compilable annotation, should be in file package-info.java
+@Deprecated
+package org.checkstyle.suppressionxpathfilter.packageannotation; // warn
```

Comment:
there is no reason to work on non-compilable files, we do not support this.

please create `package-info.java` and do validation on it